Output.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace dokuwiki\plugin\qc;
+
+/**
+ * Class Output
+ *
+ * Create the HTML formatted output of the scoring analysis
+ *
+ * @package dokuwiki\plugin\qc
+ */
+class Output {
+
+    const MAXERR = 10; //what score to use as total failure
+
+    /** @var array the scoring data */
+    protected $data;
+
+    /** @var  \helper_plugin_qc */
+    protected $helper;
+
+    /**
+     * Output constructor.
+     * @param string $page the page to analyze
+     */
+    public function __construct($page) {
+        $this->helper = plugin_load('helper', 'qc');
+        $this->data = $this->helper->getQCData($page);
+    }
+
+    /**
+     * Get the score as icon
+     *
+     * @param $score
+     * @return string
+     */
+    public static function scoreIcon($score) {
+        $html = '';
+
+        // rate the score
+        if($score > self::MAXERR) {
+            $rating = 'bad';
+        } elseif($score) {
+            $rating = 'meh';
+        } else {
+            $rating = 'good';
+        }
+
+        // output icon and score
+        $html .= '<span class="qc_icon qc_' . $rating . '">';
+        $html .= inlineSVG(__DIR__ . '/svg/' . $rating . '.svg');
+        if($score) $html .= '<span>' . $score . '</span>';
```

Comment:
Hmm it's the only span inside .qc_icon. It's adressable via the cascade already. I don't see why a class would be necessary.